crypto/evp/evp_enc.c

```diff
@@ -631,8 +630,8 @@ int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
         return 0;
```

Comment:
In the code above the ctx->cipher->block_size is int. Perhaps use that in the condition and only after that assign the number to size_t blocksize?